resource/StandardResources.md

```diff
@@ -0,0 +1,94 @@
+# Standard Resources
+
+This page lists the standard resource types in OpenCensus. For more details on how resources can 
+be combined see [this](Resource.md).
+
+OpenCensus defines these fields.
+ * [Compute Unit](#compute-unit)
+   * [Container](#container)
+ * [Deployment Service](#deployment-service)
+   * [Kubernetes](#kubernetes)
+ * [Compute Instance](#compute-instance)
+   * [Host](#host)
+ * [Environment](#environment)
+   * [Cloud](#cloud)
+   * [Cluster](#cluster)
+
+## TODOs
```

Comment:
another TODO - what does web should report from the browser?

Reply:
... and later from devices.

Reply:
Done.